### PR TITLE
Add back popup dialogs for tabs U.I.

### PR DESCRIPTION
**Pull request by sjpadgett**
I believe popup menu items are best served with dialogs as they are legacy, however; may have been intentional to place in tabs and if so, can add a global to enable such. I do hate adding globals tho ...

### CODE REVIEW

File: interface/main/tabs/js/tabs_view_model.js [hunk 1]
```diff
@@ -35,8 +35,8 @@ function tabStatus(title,url,name,closable,visible,locked)
 function tabs_view_model()
 {
     this.tabsList=ko.observableArray();
-    this.tabsList.push(new tabStatus("One",webroot_url+"/interface/main/main_info.php","cal",true,true,false));
-    this.tabsList.push(new tabStatus("Two",webroot_url+"/interface/main/messages/messages.php?form_active=1","msg",true,false,false));
+    this.tabsList.push(new tabStatus("Loading...",webroot_url+"/interface/main/main_info.php","cal",true,true,false));
+    this.tabsList.push(new tabStatus("Loading...",webroot_url+"/interface/main/messages/messages.php?form_active=1","msg",true,false,false));
```
Comment (bradymiller) [inline interface/main/tabs/js/tabs_view_model.js]: Nice! That one, two thing was weird. :)

**Comment by bradymiller** [comment]
If tests well for you, then rec bringing it in. If you want more testing, though, just let me know (and then I'll start up a testing docker).

**Comment by sjpadgett** [comment]
No docker needed but I will keep open a couple days because a couple of the popup scripts are broken in both UI versions and i'll take a wack at them here and there. Appointments has redirect to superbill and address labels which likely never worked or I don't get it plus you know me, I can't seem to mind my business. A curse, I tell ya.. A curse :)

File: interface/main/tabs/js/tabs_view_model.js (hunk 1 above)
Comment (sjpadgett) [reply interface/main/tabs/js/tabs_view_model.js]: Was driving me crazy. I kept expecting three:)

**Comment by sjpadgett** [comment]
Going to bring this in before it get's stale. Will circle back to payments later.

**Pull request merged**